packages/auth/src/authClients/clerk.ts

```diff
@@ -22,9 +26,9 @@ export type ClerkUser = ClerkUserResource & { roles: string[] | null }
 // In production, there is an issue where the AuthProvider sometimes captures
 // Clerk as null. This intercepts that
 // issue and falls back to `window.Clerk` to access the client.
-function clerkClient(propsClient: Clerk | null) {
-  if (!propsClient) {
-    return window.Clerk ?? null
+function clerkClient(propsClient: Clerk | null): Clerk | null {
+  if (!propsClient && typeof window !== undefined) {
+    return (window as any).Clerk ?? null
```

Comment:
@zackdotcomputer Not really part of this PR, but do you know when this is needed? The comments above are pretty vague with "an issue ... sometimes". Have you seen this happen? When/how?

Reply:
I do now - I've added more context.

Reply:
Thanks for looking in to this